framework-plugins/lisk-framework-faucet-plugin/src/plugin/schemas.ts

```diff
@@ -60,11 +65,11 @@ export const configSchema = {
 			description: 'The re-captcha site key',
 		},
 	},
-	required: ['encryptedPrivateKey', 'captchaSecretkey', 'captchaSitekey'],
+	required: ['tokenID', 'encryptedPrivateKey', 'captchaSecretkey', 'captchaSitekey'],
 	default: {
 		port: 4004,
 		host: '127.0.0.1',
-		applicationUrl: 'ws://localhost:8080/ws',
+		applicationUrl: 'ws://localhost:7887/rpc-ws',
```

Comment:
will this port be a problem when its testnet? if we have different port for beta and testnet then its best to read it dynamically?

Reply:
this is default value, so it can be set from config